src/ERC20Reserve.sol

```diff
@@ -29,23 +37,35 @@ contract ERC20Reserve is IERC20Reserve, Ownable {
         _;
     }
 
-    function withdraw(IERC20 token, uint256 amt) public override onlyUser {
+    function withdraw(
+        IERC20 token,
+        address to,
+        uint256 amt
+    ) public override onlyUser {
         uint256 balance = withdrawable[token];
         require(balance >= amt, "Reserve: withdrawal over balance");
         withdrawable[token] = balance - amt;
-        require(token.transfer(msg.sender, amt), "Reserve: transfer failed");
-        emit Withdrawn(msg.sender, token, amt);
+        require(token.transfer(to, amt), "Reserve: transfer failed");
+        emit Withdrawn(msg.sender, token, to, amt);
```

Comment:
```suggestion
        emit Withdraw(msg.sender, token, to, amt);
```

Reply:
I think present tense is nicer for the event names

Reply:
FWIW OpenZeppelin consistently uses past tense. ERCs sometimes use past tenses as well (e.g. 777) or nouns derived from verbs (e.g. 20, 721, 1155). IMO if we want to use verbs, we probably should stick to the past forms to lower the surprise factor :shrug: